build.gradle

```diff
@@ -123,14 +136,9 @@ project('common') {
 
     javadoc {
         title = "Pravega Common Libraries"
+        dependsOn delombok
```

Comment:
Isn't there a way in gradle to apply these to all projects? Can we use the `allprojects` section at the top of this file?

Reply:
Tried a couple of approaches, do not see a way to ensure tasktype `javadoc` to depend on delombok by default.